src/ildasm/dasm.cpp

```diff
@@ -7873,12 +7877,13 @@ BOOL DumpFile()
                     {   // initialized data
                         szptr+=sprintf_s(szptr,SZSTRING_REMAINING_SIZE(szptr),"%s%8.8X = %s (",szTls,fromPtr,KEYWORD("bytearray"));
                         printLine(g_pFile,szString);
-                        sprintf_s(szString,SZSTRING_REMAINING_SIZE(szptr),"%s                ",g_szAsmCodeIndent);
+                        szptr = szString;
+                        sprintf_s(szptr,SZSTRING_REMAINING_SIZE(szptr),"%s                ",g_szAsmCodeIndent);
```

Comment:
My understanding is that basically we flush szString in ```printLine``` above, we can start using szString from the beginning. So, I think the following change would suffice instead of assigning szString to szPtr and operating on it.
```
sprintf_s(szString, SZSTRING_SIZE,"%s",g_szAsmCodeIndent);
```